src/routes/view-content/[guideId]/[bibleSection]/MainMenu.svelte

```diff
@@ -0,0 +1,97 @@
+<script lang="ts">
+    import { goto } from '$app/navigation';
+    import { _ as translate } from 'svelte-i18n';
+    import DownloadIcon from '$lib/icons/DownloadIcon.svelte';
+    import HomeIcon from '$lib/icons/HomeIcon.svelte';
+    import LanguageIcon from '$lib/icons/LanguageIcon.svelte';
+    import ShareIcon from '$lib/icons/ShareIcon.svelte';
+    import CogIcon from '$lib/icons/CogIcon.svelte';
+    import ChatBubbleIcon from '$lib/icons/ChatBubbleIcon.svelte';
+    import Feedback from './feedback/Feedback.svelte';
+    import SettingsMenu from './settings-menu/SettingsMenu.svelte';
+    import QuickShare from './quick-share/QuickShare.svelte';
+
+    enum MenuEnum {
+        settings = 'settings',
+        share = 'share',
+        feedback = 'feedback',
+    }
+
+    let currentMenu: MenuEnum | null = null;
+</script>
+
+{#if currentMenu === MenuEnum.settings}
+    <SettingsMenu close={() => (currentMenu = null)} />
+{:else if currentMenu === MenuEnum.share}
+    <QuickShare close={() => (currentMenu = null)} />
+{:else if currentMenu === MenuEnum.feedback}
+    <Feedback close={() => (currentMenu = null)} />
```

Comment:
Instead of handling the menus all at the top level, the MainMenu now renders and controls the sub-menus. Less going on in the +page.svelte.